talend-component-maven-plugin/src/it/dependency/pom.xml

```diff
@@ -84,17 +82,32 @@
         <groupId>org.talend.sdk.component</groupId>
         <artifactId>talend-component-maven-plugin</artifactId>
         <version>@project.version@</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.talend.components</groupId>
+            <artifactId>@it-component-artifact-id@</artifactId>
+            <version>@it-component-version@</version>
+            <classifier>component</classifier>
+            <type>car</type>
+          </dependency>
+        </dependencies>
         <executions>
           <execution>
             <id>generate-input</id>
-            <phase>generate-sources</phase>
             <goals>
               <goal>dependencies</goal>
             </goals>
+            <phase>generate-sources</phase>
+          </execution>
+          <execution>
+            <id>prepare-repository</id>
+            <goals>
+              <goal>prepare-repository</goal>
+            </goals>
+            <phase>test</phase>
```

Comment:
Not sure adding a new phase here is the right way but adding a new pom might be worse... 
I'll let you decide.

Reply:
Fine for me